src/routes/users.test.js

```diff
@@ -476,27 +471,117 @@ test("Users can't change protected fields using PATCH", async t => {
 });
 
 test('Users can change allowed fields', async t => {
-    let user = await t.context.getUser();
+    let { user, session } = await t.context.getUser();
+
+    const oldEmail = user.email;
 
     const allowedFields = {
         name: 'My new name',
-        email: 'new@example.com'
+        email: `new-${Math.round(Math.random() * 1e6).toString(36)}@example.com`,
```

Comment:
```suggestion
        email: t.context.getCredentials().email
```

This will create a unique new email address like in other tests.